staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo_test.go

```diff
@@ -45,7 +45,9 @@ func TestGetAPIRequestInfo(t *testing.T) {
 
 		// resource paths
 		{"GET", "/api/v1/namespaces", "list", "api", "", "v1", "", "namespaces", "", "", []string{"namespaces"}},
-		{"GET", "/api/v1/namespaces/other", "get", "api", "", "v1", "other", "namespaces", "", "other", []string{"namespaces", "other"}},
+		{"GET", "/api/v1/namespaces/other", "get", "api", "", "v1", "", "namespaces", "", "other", []string{"namespaces", "other"}},
```

Comment:
Any request to any URL prefixed with a specific namespace (e.g. `/api/$version/namespaces/$namespace`) is considered to have a namespace. That has been the case from the very beginning of Kubernetes. Changing that would break many many policies and authorization rules, so would not be permitted to change.

Reply:
Got it.